example/builder/src/pre_dart_two_nine_syntax/function_component.dart

```diff
@@ -0,0 +1,14 @@
+import 'package:over_react/over_react.dart';
+
+part 'function_component.over_react.g.dart'; // ignore: uri_has_not_been_generated
+
+UiFactory<FunctionComponentProps> FunctionComponent = uiFunction(
+  (props) {
+    return Dom.div()(
+      'This is a pre-Dart 2.9.0 functional component.'
```

Comment:
This looks like a post-Dart 2.9.0 component since it's using the private config, not the public one

Reply:
🤦